consultation_analyser/consultations/migrations/0025_populate_existing_processing_run_slugs.py

```diff
@@ -0,0 +1,37 @@
+# Populate slug field for existing ProcessingRuns
+
+from django.db import migrations
+from wonderwords import RandomWord
+
+
+def generate_random_slug():
+    word1 = RandomWord().word().lower()
+    word2 = RandomWord().word().lower()
+    slug = f"{word1}-{word2}"
+    return slug
+
+
+def populate_processing_run_slug(apps, schema_editor):
+    ProcessingRun = apps.get_model("consultations", "ProcessingRun")
+    all_runs = ProcessingRun.objects.all()
+    number_runs = all_runs.count()
+    slugs = set()
+    while len(slugs) < number_runs:
```

Comment:
Could we just call run.save() given that that method will now create a unique slug if one is missing?

Reply:
Turns out you can't - migrations won't run custom `save()`.

I have still updated this to use the logic from the custom save and the `faker` package.